src/pages/Layout.tsx

```diff
@@ -11,7 +11,6 @@ const styles = {
     boxSizing: "border-box",
     flex: "0 0 350px",
     minWidth: 0,
-    paddingLeft: "40px",
```

Comment:
Can we bring this back? It saves us from having to add padding individually to whatever the right content is